src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -450,7 +450,7 @@ def __get_filters(self):
             self._is_save_cache = False
             cached_test_list = self.__prepare_smart_filters(cached_test_dict)
         runtime_test_list = list()
-        if len(runtime_test_dist) > 0:
+        if len(runtime_test_dist) > 0 and self._split_unit == constants.SUITE_UNIT_NAME:
```

Comment:
Cache is also using for functional tests where suite is base split unit. Should we remove `and self._split_unit == constants.SUITE_UNIT_NAME`? In other case please explain the details